x-pack/elastic-agent/docs/elastic-agent-command-line.asciidoc

```diff
@@ -1,8 +1,12 @@
 [[elastic-agent-cmd-options]]
+[role="xpack"]
 = Command line options
 
 experimental[]
 
+//REVIEWERS: There are a couple of options missing here: strict.perms and
```

Comment:
I'm going to remove this comment and follow up separately so we can get this merged.